package.json

```diff
@@ -39,7 +39,6 @@
     "eslint-plugin-import": "2.3.0",
     "flow-bin": "0.38.0",
     "jasmine-core": "2.5.2",
-    "jshint": "2.9.4",
```

Comment:
I noticed some inline jshint disabling still left behind. Maybe it's good to just remove those now?